11_getTheTitles/getTheTitles.js

```diff
@@ -1,6 +1,11 @@
-const getTheTitles = function() {
+const getTheTitles = function (arr) {
+    let filtered = arr.map(books => (books.title))
+    return filtered
+
```

Comment:
one line statement,

could just reduce the hole function getTheTitle to a oneliner